frontends/web/src/components/wait-dialog/wait-dialog.jsx

```diff
@@ -61,6 +61,7 @@ export default class WaitDialog extends Component {
         active,
     }) {
         const isActive = active ? style.active : '';
+        const titleText = title || 'Action needed on your BitBox02';
```

Comment:
Default title is not good. Now we have 'Action needed on your BitBox02' everywhere, for example the success message when sending a transaction using BitBox01.